bokeh/models/graphs.py

```diff
@@ -161,6 +162,19 @@ class EdgesAndLinkedNodes(GraphHitTestPolicy):
 
     pass
 
+class NodesAndAdjacentNodes(GraphHitTestPolicy):
+    '''
+    With the ``NodesAndAdjacentNodes`` policy, inspection or selection of
+    graph nodes will result in the inspection or selection of the node and
+    those directly connected to it by edges. There is no direct selection
+    or inspection of graph edges or nodes. The last entry in each list
+    returned by NodesAndAdjacentNodes refers to the original node that was
+    inspected or selected.
```

Comment:
Can we contrive it to always be the first, instead? I think that would be simpler for users to make use of.

Reply:
Initially I had set the selected node to be the first in the list. However, I noticed in bokeh/examples/plotting/file/graphs.py that when the hover tool is set to print selection data in the graphic, it prints the data over the last node in the selection list. See the attached video for reference. (https://user-images.githubusercontent.com/90462878/145235745-1f37ce8a-7f92-40e2-bcec-ec42fb719a4a.mov).

I'm not too familiar with the hover tool or the mechanisms behind it, but if there is a way to make it use the first element instead of the last as an indicator of where to print selection data, then I'd be happy to make this change!

Reply:
Oh I see, I am not sure we have ever actually made any specific guarantees about the order. If it is not something that is documented and under test, this its just be undefined behavior that happens to be useful. I think it would be safest to just remove the claim here altogether for now, and make a separate issues about finding a way to communicate the "tool selected" nodes vs the "policy selected" node. Then a PR can consider all the Polices together (to take a consistent approach) and also make sure everything is documented together and maintained under test.

Reply:
Sure, I'll edit the claim accordingly in the policy description!